.github/workflows/percy-snapshots.yml

```diff
@@ -7,9 +7,10 @@ on:
 
 jobs:
```

Comment:
would you mind adding `features/*` to the branches list in https://github.com/yext/search-ui-react/blob/544236fa3042725affa356dc5f2cad9406d19f35/.github/workflows/run-linting.yml